.buildkite/scripts/steps/webpack_bundle_analyzer/upload.ts

```diff
@@ -55,6 +55,7 @@ const upload = () => {
     fs.writeFileSync('index.html', html);
     console.log('--- Uploading Webpack Bundle Analyzer reports');
     exec(`
+      .buildkite/scripts/common/activate_service_account.sh gs://ci-artifacts.kibana.dev
```

Comment:
Thanks for finding this - it eluded me. I solved it now, not with setting cwd, because I think the scripts are expected to run from specific folders. I resolved a relative path from these locations. Now it seems to work:
```
Activated service account credentials for: [kibana-ci-sa-proxy@elastic-kibana-ci.iam.gserviceaccount.com]
  | Activated service account kibana-ci-sa-proxy@elastic-kibana-ci.iam.gserviceaccount.com
  | Impersonating kibana-ci-access-artifacts@elastic-kibana-ci.iam.gserviceaccount.com
  | Updated property [auth/impersonate_service_account].
  | Activated service account kibana-ci-access-artifacts@elastic-kibana-ci.iam.gserviceaccount.com
  | WARNING: This command is using service account impersonation. All API calls will be executed as [kibana-ci-access-artifacts@elastic-kibana-ci.iam.gserviceaccount.com].
```